Intro_Sort/Intro_Sort.cpp

```diff
@@ -0,0 +1,157 @@
+#include<bits/stdc++.h> 
+using namespace std; 
+
+// A utility function to swap the values pointed by 
+// the two pointers 
+void swapValue(int *a, int *b) 
+{ 
+       int *temp = a; 
+       a = b; 
+       b = temp; 
+   return; 
+} 
+
+/* Function to sort an array using insertion sort*/
+void InsertionSort(int arr[], int *begin, int *end) 
+{ 
+      // Get the left and the right index of the subarray 
+      // to be sorted 
+      int left = begin - arr; 
+      int right = end - arr; 
+
+     for (int i = left+1; i <= right; i++) 
+      { 
+      int key = arr[i]; 
+         int j = i-1; 
+
+         /* Move elements of arr[0..i-1], that are 
+       greater than key, to one position ahead 
+       of their current position */
+          while (j >= left && arr[j] > key) 
+     { 
+       arr[j+1] = arr[j]; 
+       j = j-1; 
+      } 
+           arr[j+1] = key; 
+} 
+
+  return; 
+} 
+
+    // A function to partition the array and return 
+    // the partition point 
+   int* Partition(int arr[], int low, int high) 
+{ 
+     int pivot = arr[high];  // pivot         
+     int i = (low - 1); // Index of smaller element 
+
+    for (int j = low; j <= high- 1; j++) 
+      { 
+         // If current element is smaller than or 
+        // equal to pivot 
+         if (arr[j] <= pivot) 
+         {
+          // increment index of smaller element 
+   i++; 
+
```

Comment:
remove this line

Reply:
the comment line or the i++ ? @atarax665